packages/fxa-settings/src/lib/utilities.ts

```diff
@@ -100,3 +100,9 @@ export function isMobileDevice(client?: AttachedClient) {
   }
   return /mobi/i.test(navigator.userAgent);
 }
+
+// Crockford base32 Regex. Case insensitive and excludes I, L, O, U
```

Comment:
We may want to move this to some new validation file, but it seemed best here until we have others like it.